modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2AggregateKeyDXP.testcase

```diff
@@ -0,0 +1,171 @@
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+		}
+
+		task ("Create a new Customer Portal Site") {
+			CustomerPortal.addCustomerPortalSite(siteName = "${testSiteName}");
+		}
+
+		task ("Add test user access to all Accounts") {
+			CustomerPortal.assignUserToOrganization();
+
+			CustomerPortal.addUserToOneOrMoreAccounts(accountsList = "${testAccount}");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		JSONGroup.deleteGroupByName(groupName = "${testSiteName}");
+
+		CustomerPortal.customerPortalApisTearDown();
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Automation ID: LPS-148069 | Test Summary: Assert the name of the downloaded file follows the standard"
+	@priority = "3"
+	test AssertAggregateActivationKeyXmlFileIsDownloaded {
+
+		// TODO LPS-148069 AssertAggregateActivationKeyXmlFileIsDownloaded pending implementation
+
+	}
+
+	@description = "Automation ID: LPS-148070 | Test Summary: Verify success message Activation Key was downloaded successfully"
+	@priority = "4"
+	test CanViewConfirmSuccessfulMessage {
+		property portal.upstream = "quarantine";
+
+		// Needs refactor since it depends on example FIFA account
+
+		task ("Given a user signs in on OKTA to update the Customer Portal") {
+			CustomerPortal.loginOktaToUpdateCP();
+
+			Navigator.openURL();
+
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+		}
+
+		task ("Given I go to the test account") {
+			CustomerPortal.search(searchItem = "${testAccount}");
+
+			CustomerPortal.enterOnProject(title = "${testAccount}");
+		}
+
+		task ("And I click on DXP product") {
+			CP2NavigationMenu.goToProductActivation();
+
+			CP2NavigationMenu.clickOnProductButton(product = "${testProduct}");
+		}
+
+		task ("I selected two keys in the table") {
+			CP2ActivationKeys.selectKeyCheckboxByExpirationDate(
+				date = "Mar 30, 2023",
+				position = "1");
+
+			CP2ActivationKeys.selectKeyCheckboxByExpirationDate(
+				date = "Mar 30, 2023",
+				position = "3");
+		}
+
+		task ("When I click at the 'Aggregate key single file' option") {
+			CustomerPortal.clickAtButton(text = "download");
+
+			CP2ActivationKeys.clickAtDropdownItem(dropdownItem = "Aggregate key single file");
+		}
+
+		task ("Then I assert that the alert with the message 'Activation Key Was Downloaded Successfully' is visible") {
+			CustomerPortal.viewAlert(alertText = "Activation Key was downloaded successfully");
+		}
+
+		//task ("And I get the expected downloaded file name") {
+		//	var downloadedFileName = CP2ActivationKeys.formatLicenseFileName(
+		//		fileType = "xml",
+		//		licenseName = "${testAccount}",
+		//		licenseType = "activation-key-multiple-products",
+		//		version = "7.4");
+		//}
+		//task ("And I assert that the Activation Key was downloaded") {
+		//	CP2FileUtil.assertDownloadedFileIsPresent(downloadedFileName = "${downloadedFileName}");
+		//}
```

Comment:
Is there a reason why we have this part commented, since this test is in quarantine?

Reply:
Done